src/Button/Button.Component.js

```diff
@@ -110,6 +110,22 @@ const clickBtnHandler = btn => {
         alert(`You clicked the ${btn} Button`);
     };
 
+    const buttonOptionsDescription = `
+There are three emphasis styles used to indicate the importance of the button on the page. \n\n
+* **Emphasized Button**: There should only be one highlighted button on the page. This is
+the primary call to action.\n\n
+* **Regular Button**: The default button style and the most common button. There may be
```

Comment:
@mobot11 Is `shell` now the same as the regular/default option?

Reply:
I think `shell` is leftover from some previous iterations.  Adding `option='shell'` _does_ do some alternate styling, but the button is very faint and does not look usable.  Furthermore, the SAP Fiori Fundamentals site shows only three options for Button:  **Emphasized**, **Regular** and **Light** and the component is marked "Stable" (see screenshot below).  I think we should remove `shell` from the list of available options for a button.

![screen shot 2019-02-06 at 10 16 29 am](https://user-images.githubusercontent.com/11407353/52355830-b661c600-29f8-11e9-9b5b-bd758e60c3b8.png)

Reply:
I think we're good to remove `shell` as an option.